packages/edit-donation/src/donation-form-edit-donation.ts

```diff
@@ -109,9 +111,27 @@ export class DonationFormEditDonation extends LitElement {
   }
 
   updated(changedProperties: PropertyValues): void {
+    if (changedProperties.has('customAmountSelected')) {
+      this.customAmountButton.checked = this.customAmountSelected;
+    }
+    if (changedProperties.has('amountOptions')) {
+      this.customAmountSelected = false;
+      this.updateSelectedDonationInfo();
+      this.setupAmountColumnsLayoutConfig();
+    }
     if (changedProperties.has('donationInfo')) {
       this.updateSelectedDonationInfo();
     }
+    if (
+      changedProperties.has('defaultSelectedAmount') &&
+      this.defaultSelectedAmount
```

Comment:
QUESTION: Is there a distinction needed between a `defaultSelectedAmount` of `0` and some other false-y value?

Reply:
Ah good question.. I suppose `0` could be a default selected value and then it wouldn't get selected 😅 I'll update that check